_pages/presentation_materials.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: page
+title: "Apache Beam Presentation Materials"
+permalink: /presentation-materials/
+---
+
+Are you interested in learning more about Apache Beam or giving a talk about Apache Beam? Excellent! The Apache Beam community has created this collection of materials to help you get started. The community periodically reviews these materials to ensure they are accurate and up-to-date.
+
+<div class="panel panel-default">
+  <div class="panel-body">
+    <iframe src="https://drive.google.com/embeddedfolderview?id=0B-IhJZh9Ab52a3JLVXFWMDltcHM#list" width="700" height="300" frameborder="0"></iframe>
+  </div>
+</div>
+
+### Using these materials
+These materials are free to use! If you do use these materials, please remember to give credit to the authors of the original materials. These materials include speaker notes and you can view them by clicking **`View`** and selecting **`Show speaker notes`**. A set of presentations are have `[Long]` or `[Short]` in their name. Some topics, such as the *Apache Beam Model*, have two different presentations (a long and short version) depending on your needs and time constraints.
```

Comment:
Ack - what about "please remember to give credit to the authors cited in each presentation"